azure-functions/number-generator-function/testing/defaultContext.js

```diff
@@ -0,0 +1,6 @@
+module.exports = {
+  log: jest.fn(),
+  bindingData: {
```

Comment:
Is this needed? I thought we didn't have any tests?

Reply:
Yes, it will help us to build tests on top of this. We currently do not have a test at the moment.